ocaml/src/alba_base_client.ml

```diff
@@ -41,6 +41,7 @@ class client
     ~partial_osd_read
     ~cache_on_read ~cache_on_write
     ~populate_osds_info_cache
+    ~(read_preference: string list option)
```

Comment:
because `Alba_client.make_client has a `?read_reference` parameter.  I'll suppress the option.